types/itowns/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for itowns 2.39
+// Project: https://itowns.github.io/
+// Definitions by: Bouillaguet Quentin <https://github.com/desplandis>
+//                 Madec Germerie <https://github.com/mgermerie>
```

Comment:
you'll have to get this person explicit ACK here, to be listed as an owner

Reply:
@peterblazejewicz Is this sufficient to ACK in this PR thread or shall he commit itself to the list of definitions owners?

Reply:
it's enough to read ack here on the thread somwhere, thanks! Iv'e seen emoji reaction, but that's not the same as ack imo

Reply:
Hi @peterblazejewicz ! Thanks for your review. I hereby give my ack to be listed as an owner of the package.